p256/src/ecdsa.rs

```diff
@@ -39,10 +39,12 @@
 //! # }
 //! ```
 
-pub use ecdsa_core::signature::{self, Error};
+pub use ecdsa_core::{
+    EcdsaCurve, RecoveryId,
+    signature::{self, Error},
+};
```

Comment:
This also makes `EcdsaCurve` public as I copied the block from k256. Is this a good thing? Or should I remove the `EcdsaCurve` change?